spring-cloud-stream-integration-tests/src/test/java/org/springframework/cloud/stream/config/MessageChannelConfigurerTests.java

```diff
@@ -78,9 +77,6 @@ public void testMessageConverterConfigurer() throws Exception {
 		MessageHandler messageHandler = new MessageHandler() {
 			@Override
 			public void handleMessage(Message<?> message) throws MessagingException {
-				assertThat(message.getPayload()).isInstanceOf(Tuple.class);
-				assertThat(((Tuple) message.getPayload()).getFieldNames().get(0)).isEqualTo("message");
-				assertThat(((Tuple) message.getPayload()).getValue(0)).isEqualTo("Hi");
```

Comment:
This test is perhaps irrelevant now. We no longer try to convert messages based on payload, and since there's no more serializeIfNecessary step, when you subscribe to an input channel directly, you get the original message written to the channel.

I'll revisit this, as the right test now should be putting a payload of `byte[]` into the channel, because that's what the Binders will offer anyways.